seq2seq/third_party/utils/__init__.py

```diff
@@ -0,0 +1,11 @@
+from .utils import (
```

Comment:
I assume you wrote this file from scratch? Can you just confirm that? If that is true, it is fine, but if not please link me to the original version.

Reply:
I wrote it